### PR TITLE
i18n(zh-cn): Translate `guides/integrations-guide/vue.mdx & solid-js.mdx`

### PR DESCRIPTION
**Pull request by XinChou16**
#### What kind of changes does this PR include?
- New content
- Translated content

#### Description
- Translate following mdx content to Chinese.
	- **`guides/integrations-guide/vue.mdx`**
	- **`guides/integrations-guide/solid-js.mdx`**

### CODE REVIEW

File: src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+此 **[Astro 集成][astro-integration]** 为你的 [SolidJS](https://www.solidjs.com/) 组件启用服务器端渲染和客户端水合。
+
+## 安装
+
+有两种方式可以将集成添加到你的项目当中，先让我们从最方便的方式来入手！
+
+### `astro add` 命令
+
+Astro 包含来一个 CLI 工具来提供一等集成支持，`astro add` 这个命令将会：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx:19]:
```suggestion
Astro 包含一个 CLI 工具来提供一等集成支持，`astro add` 这个命令将会：
```

File: src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+要在 Astro 使用你的 SolidJS 组件，你可以移步我们的 [UI 框架文档][astro-ui-frameworks]. 你将会了解到：
+
+*   📦 如何加载框架组件
+*   💧 客户端水合选项
+*   🤝 将框架混合和嵌套在一起的时机
+
+## 疑难解答
+
+如需帮助， 请查看我们在 [Discord](https://astro.build/chat) 上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx:74]:
```suggestion
如需帮助，请查看我们在 [Discord](https://astro.build/chat) 上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```

File: src/content/docs/zh-cn/guides/integrations-guide/vue.mdx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+此 **[Astro 集成][astro-integration]** 为你的 [Vue 3](https://vuejs.org/) 组件启用服务器端渲染和客户端水合。
+
+## 安装
+
+有两种方式可以将集成添加到你的项目当中，先让我们从最方便的方式来入手！
+
+### `astro add` 命令
+
+Astro 包含来一个 CLI 工具来提供一等集成支持，`astro add` 这个命令将会：
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/vue.mdx:19]:
```suggestion
Astro 包含一个 CLI 工具来提供一等集成支持，`astro add` 这个命令将会：
```

File: src/content/docs/zh-cn/guides/integrations-guide/vue.mdx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+要在 Astro 使用你的 Vue 组件，你可以移步我们的 [UI 框架文档][astro-ui-frameworks]。 你将会了解到：
+
+*   📦 如何加载框架组件
+*   💧 客户端合成注水选项
+*   🤝 将框架混合和嵌套在一起的时机
+
+## 疑难解答
+
+如需帮助， 请查看我们在 [Discord](https://astro.build/chat) 上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/vue.mdx:74]:
```suggestion
如需帮助，请查看我们在 [Discord](https://astro.build/chat) 上的  `#support` 频道。 我们友好的支持小队成员随时为你提供帮助!
```

File: src/content/docs/zh-cn/guides/integrations-guide/vue.mdx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+
+该 Astro 包是由核心团队维护的，欢迎提交 issue 和 PR！
+
+[astro-integration]: /zh-cn/guides/integrations-guide/
+
+[astro-ui-frameworks]: /zh-cn/core-concepts/framework-components/#使用框架组件
+
+## 选项
+
+此集成由 `@vitejs/plugin-vue` 提供支持。 要定制 Vue 编译器, 你可以为集成提供选项。 查看 `@vitejs/plugin-vue` [文档](https://www.npmjs.com/package/@vitejs/plugin-vue) 获取更多详细内容。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/vue.mdx:88]:
```suggestion
此集成由 `@vitejs/plugin-vue` 提供支持。要定制 Vue 编译器, 你可以为集成提供选项。查看 `@vitejs/plugin-vue` [文档](https://www.npmjs.com/package/@vitejs/plugin-vue) 获取更多详细内容。
```

File: src/content/docs/zh-cn/guides/integrations-guide/vue.mdx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+    }),
+  ],
+});
+```
+
+### appEntrypoint
+
+你可以拓展 Vue `app` 实例并将 `appEntrypoint` 选项设置为一个相对根目录的导入标识符 (例如： `appEntrypoint: "/src/pages/_app"`)。
+
+此文件的默认导出应该是一个接收 Vue `App` 实例的函数, 允许使用 [自定义 Vue 插件](https://vuejs.org/guide/reusability/plugins.html), `app.use` 和其他高级使用情形的定制。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/vue.mdx:115]:
```suggestion
此文件的默认导出应该是一个接收 Vue `App` 实例的函数，允许使用 [自定义 Vue 插件](https://vuejs.org/guide/reusability/plugins.html)，`app.use` 和其他高级使用情形的定制。
```

File: src/content/docs/zh-cn/guides/integrations-guide/vue.mdx [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import vue from '@astrojs/vue';
+
+export default defineConfig({
+  integrations: [vue({ jsx: true })],
+});
+```
+
+这将会为 Vue 和 Vue JSX 组件 开启渲染，要定制 Vue JSX 编译器的话, 可以把传递的选项由布尔值改为对象，查阅 `@vitejs/plugin-vue-jsx` [文档](https://www.npmjs.com/package/@vitejs/plugin-vue-jsx) 获取更多细节内容。
```
Comment (100gle) [inline src/content/docs/zh-cn/guides/integrations-guide/vue.mdx:151]:
```suggestion
这将会为 Vue 和 Vue JSX 组件 开启渲染，要定制 Vue JSX 编译器的话，可以把传递的选项由布尔值改为对象，查阅 `@vitejs/plugin-vue-jsx` [文档](https://www.npmjs.com/package/@vitejs/plugin-vue-jsx) 获取更多细节内容。
```

**Review by 100gle — CHANGES_REQUESTED** [review]
需要稍微注意一下语句是否通顺和排版

**Comment by 100gle** [comment]
LGTM!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**